pyvista/plotting/colors.py

```diff
@@ -1204,25 +1208,6 @@ def get_cmap_safe(cmap):
             except AttributeError:
                 pass
 
-        # Else use Matplotlib
-        if not has_module('matplotlib'):
-            raise ImportError(
-                'The use of custom colormaps requires the installation of matplotlib.'
-            )  # pragma: no cover
-
-        try:
-            from matplotlib import colormaps, colors
-        except ImportError:  # pragma: no cover
-            import matplotlib
-            import scooby
-
-            min_req = '3.5.0'
-            if not scooby.meets_version(matplotlib.__version__, min_req):
-                raise ImportError(
-                    'The use of custom colormaps requires the installation of '
-                    f'matplotlib>={min_req}.'
-                ) from None
```

Comment:
I wonder about this. [3.5.0 is a bit more than a year old](https://github.com/matplotlib/matplotlib/releases/tag/v3.5.0). So
1. We shouldn't assume that users have it.
2. Is it really necessary for custom colormaps? This sounds very suspicious; if this is indeed a requirement then I'd be tempted to go out of our way to get around this constraint. @akaszynski do you remember what this was about?

Reply:
`<=3.5.0` This would break mathtex/latex support from recent versions of VTK

Reply:
We should permit `<=3.5.0` and not make it a hard requirement. However, to @banesullivan's point, we need to add a runtime warning/error when mathtex or latex is used and an older version of `matplotlib` is installed.

Since this is already an issue, I'm going to call it out of scope of this PR and recommend a follow-up.